megamek/src/megamek/common/QuirkEntry.java

```diff
@@ -90,7 +90,7 @@ public QuirkEntry(String code, String location, int slot, String weaponName, Str
             throw new IllegalArgumentException("No location for " + code + " : " + unitId);
         } else if (StringUtility.isNullOrBlank(weaponName)) {
             throw new IllegalArgumentException("No weapon for " + code + " : " + unitId);
-        } else if ((slot < 0) || (slot > 11)) {
+        } else if (slot < 0) {
```

Comment:
Maybe "slot > maximum number of slots in the location"? Or does it not really matter?

Reply:
I would need the entity to do that. QuirkEntry is just a helper for these unit file lines: weaponquirk:stable_weapon:RT:3:ISGaussRifle
I'd say it's not such a big deal. (And I think it's more likely that the slot number is ok but there's no weapon there :D)

Reply:
Fair enough